tests/passw_hardening_test.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
```

Comment:
Why do we need tests for auto-generated CLI? Is it because of the coverage?

Reply:
yes